doc/sphinx/source/project/extras.rst

```diff
@@ -20,9 +30,60 @@ Homomorphic Encryption (HE)
 
 
 
+.. _distributed_training:
+
+Distributed training with nGraph
+================================
+
+.. important:: Distributed training is not officially supported as of version
+   |version|; however, some configuration options have worked for nGraph 
+   devices in testing environments.
+
+
+How? (Generic frameworks)
+-------------------------
+
+See also: :doc:`../core/constructing-graphs/distribute-train`
+
+To synchronize gradients across all workers, the essential operation for data
+parallel training, due to its simplicity and scalability over parameter servers,
+is ``allreduce``. The AllReduce op is one of the nGraph Library’s core ops. To
+enable gradient synchronization for a network, we simply inject the AllReduce op
+into the computation graph, connecting the graph for the autodiff computation
+and optimizer update (which then becomes part of the nGraph graph). The
+nGraph Backend will handle the rest.
+
+Data scientists with locally-scalable rack or cloud-based resources will likely
+find it worthwhile to experiment with different modes or variations of
+distributed training. Deployments using nGraph Library with supported backends
+can be configured to train with data parallelism and will soon work with model
+parallelism. Distributing workloads is increasingly important, as more data and
+bigger models mean the ability to :doc:`../core/constructing-graphs/distribute-train`
+work with larger and larger datasets, or to work with models having many layers
+that aren't designed to fit to a single device.
+
+Distributed training with data parallelism splits the data and each worker
+node has the same model; during each iteration, the gradients are aggregated
+across all workers with an op that performs "allreduce", and applied to update
+the weights.
+
+Using multiple machines helps to scale and speed up deep learning. With large 
+mini-batch training, one could train ResNet-50 with Imagenet-1k data to the
+*Top 5* classifier in minutes using thousands of CPU nodes. See
+`arxiv.org/abs/1709.05011`_.
+
+
+Future work
+===========
+
+More communication ops support is in the works. See also:  
+:doc:`../../core/passes/list-of-passes`. 
```

Comment:
Not sure why we'd link to this here

Reply:
Something YinYin and I documented quite a long time ago, from when we had a larger section on "distributed" training... I think this might not be relevant with the new organization.

Reply:
Right, I think it's not relevant unless we're calling out some particular pass in the context of distributed training (in which case we should probably specify what pass we are referring to)

Reply:
Done